src/zmq/zmqpublishnotifier.cpp

```diff
@@ -211,3 +214,23 @@ bool CZMQPublishRawTransactionLockNotifier::NotifyTransactionLock(const CTransac
     ss << transaction;
     return SendMessage(MSG_RAWTXLOCK, &(*ss.begin()), ss.size());
 }
+
+bool CZMQPublishHashGovernanceVoteNotifier::NotifyGovernanceVote(const CGovernanceVote &vote)
+{
+    uint256 hash = vote.GetHash();
+    LogPrint("zmq", "zmq: Publish governancevote %s\n", hash.GetHex());
+    char data[32];
+    for (unsigned int i = 0; i < 32; i++)
```

Comment:
I see you're using the same format as above, but wondering if there's a way to use existing serialization methods for this. Also wondering if we can move `hash.begin()` out of the loop and only call it once.